src/main/resources/reference.conf

```diff
@@ -253,27 +250,27 @@ resourceTypes = {
       "own" = {
         description = "perform owner actions on the resource"
       }
-      "manage_private_user" = {
-        description = "assign, remove, and reassign the private user"
-      }
     }
     ownerRoleName = "owner"
     roles = {
+      # owner is always WSM SA
       owner = {
-        roleActions = ["delete", "read_policies", "share_policy::owner", "share_policy::editor", "share_policy::writer", "share_policy::reader", "own", "edit", "manage_private_user", "set_parent"]
-      }
-      assigner = {
-        roleActions = ["manage_private_user", "read_policies", "share_policy::editor", "share_policy::writer", "share_policy::reader"]
+        roleActions = ["delete", "read_policies", "share_policy::owner", "share_policy::editor", "share_policy::writer", "share_policy::reader", "own", "edit", "set_parent", "read", "write"]
       }
+      # editor is always the resource creator
       editor = {
-        roleActions = ["delete", "edit"]
+        roleActions = ["delete", "edit", "read", "write"]
       }
+      # WSM does not use these reader and writer roles
```

Comment:
Is WSM the only service using `controlled-user-private-workspace-resource`?  If so and you're not using these roles, can you just remove them?  Sam does not care if they're here or not.

Reply:
Turns out I was wrong about that, so I am going to leave them in.